compliant-reward-distribution/frontend/src/components/submission/Submission.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Button } from "react-bootstrap";
+import { ArrowLeft } from "lucide-react";
+import '../../styles/Submission.scss';
+import backgroundImage from "/images/Slide 16_9 - 13.svg";
+import linkImage from "/images/link.svg";
```

Comment:
```suggestion
import linkImage from "../../../public/Images/link.svg";
```

The CI pipeline fails currently, because the `yarn build` command cannot resolve the image paths.